deployment_terraform/.specific/_settings.adoc

```diff
@@ -1,5 +1,5 @@
-:quickstart-project-name: quickstart-repo-name
-:quickstart-github-org: aws-quickstart
+:partner-solution-project-name: partner-solution-repo-name
+:partner-solution-github-org: aws-partner-solution
```

Comment:
The 2nd part needs to remain as aws-quickstart at the aws-partner-solution org doesn't exist:
```suggestion
:partner-solution-github-org: aws-quickstart
```